### DISCUSSION TITLE
Bosch BTH-RM230Z cooling mode not possible and set valve status no/nc not possible

### DISCUSSION THREAD

**Discussion by metallmanu** [Koenkk/zigbee2mqtt#21009] [Devices]
Bosch BTH-RM230Z is not fully supported. The system_mode cool is missing and also to set the valves either to no (normaly open) or nc (normaly closed). ctrl_sequence_of_oper must be set from 0x02 Heating Only to 0x04 Cooling and Heating.
Also it would be nice to set the valve behaviour from no to nc

**Comment by psdl76** [comment]
I too have the problem. I can't use it to control my floor heating.

**Comment by xoxys** [comment]
>  Also it would be nice to set the valve behaviour from no to nc

Is there any way to change this at all?

**Reply by DerDreschner** [reply]
I've implemented that setting in my latest pull request, yeah.

**Comment by DerDreschner** [comment]
The missing attributes are supported with the [current 2.6.3 release](https://github.com/Koenkk/zigbee2mqtt/releases/tag/2.6.3). This discussion can be closed in my opinion.

**Comment by h1nnak** [comment]
@DerDreschner have you already seen this entry? Could you also include the new features? I know this discussion isn’t really related to it, but I didn’t want to open an issue. If I understood correctly, there won’t be any new firmware, only the new features will be released.

Release Note from February 17, 2026 [LINK](https://www.bosch-smarthome.com/uk/en/support/release-information/system-solutions/26-02-17.html)

 Room thermostat II / Room thermostat II 230 V

    Option to change the brightness setting of the device display using automations.
    Option to deactivate the humidity warning via the device’s LEDs.
    Option to activate limescale protection (only Room thermostat II 230 V).

German: [LINK](https://www.bosch-smarthome.com/de/de/support/release-informationen/system-solutions/26-02-17.html)
Raumthermostat II / Raumthermostat II 230V

    Möglichkeit zur Änderung der Helligkeitseinstellung des Displays des Geräts über Automationen.
    Möglichkeit zur Deaktivierung der Feuchtigkeitswarnung über die LEDs des Geräts.
    Möglichkeit zur Aktivierung des Verkalkungsschutzes (nur Raumthermostat II 230V).
